model/src/main/java/com/chiralbehaviors/CoRE/WellKnownObject.java

```diff
@@ -1291,6 +1291,32 @@ public String wkoName() {
             }
 
         },
+        IS_VALIDATED_BY() {
+
+            @Override
+            public String description() {
+                return "A is validated by B";
+            }
+
+            @Override
+            public WellKnownRelationship inverse() {
+                return WellKnownRelationship.VALIDATES;
```

Comment:
shouldn't this be "validates enum" or something?  Validates seems pretty broad and won't lend it self to doing different validations.